quinn-proto/src/connection/mod.rs

```diff
@@ -2943,6 +2994,22 @@ where
         self.path.sending_ecn
     }
 
+    /// Instruct the peer to retire the next `n` connection IDs, and issue replacements
+    #[cfg(test)]
+    pub(crate) fn rotate_rem_cid(&mut self, v: u64, n: u64) {
```

Comment:
```suggestion
    pub(crate) fn rotate_local_cids(&mut self, v: u64, n: u64) {
```
This is instructing the peer to change the set of CIDs it may use to address the local endpoint. We use `rem_cid` to refer CIDs issued by and used to address the peer.

Reply:
Ack. Will update